### ISSUE TITLE
config_upgrader seems to be warning me about config files in my node_modules

### ISSUE THREAD

**Issue by kjg** [codeclimate/codeclimate-eslint#181]
```
Error: /code/my_app/node_modules/cors/.eslintrc:
	Configuration for rule "quotes" is invalid:
	Severity should be one of the following: 0 = off, 1 = warn, 2 = error (you passed '"single"').
```

**Comment by kjg** [comment]
Upon further investigation it appears only top level node_modules are ignored by default.
I can work around this by adding **/node_modules/** to my .eslintignore.

Should this maybe be a default though?

**Comment by kjg** [comment]
Fixed by #182
